ui/src/app/(index)/_components/BeforeLogin.tsx

```diff
@@ -0,0 +1,29 @@
+import {Button} from '@/components/ui/button';
+import Link from 'next/link';
+import {ArrowRight} from 'lucide-react';
+import UHGroupingsInfo from '@/components/UHGroupingsInfo';
+
+const BeforeLogin = () => {
+    return (
+        <main className="bg-seafoam pb-10">
+            <UHGroupingsInfo variant={'home'}/>
```

Comment:
For any of the prop attributes where you pass a string in, you do not need the curly-braces {}. Instead, having "" is enough. For example here, `variant="home"`. Curly-braces are used for numbers and variables usually. Make sure to change this in other places in your PR too, thanks!